api-outbound.yaml

```diff
@@ -343,12 +344,16 @@ components:
         hostname:
           type: string
           description: The primary host name (local or FQDN) of the asset.
+        scanType:
```

Comment:
Q: Is there a reason why we do not use https://swagger.io/docs/specification/data-models/enums/ ?

Reply:
I didn't think about it, but I like the enum option here a lot better 👍

Reply:
Done!